### PR TITLE
Update judicial.md

### PR DESCRIPTION
**Pull request by jetfire158**

### CODE REVIEW

File: content/theme/characters/archetypes/republic/judicial.md [hunk 1]
```diff
@@ -1,6 +1,36 @@
 ---
 title: "Judicial"
 date: 2020-11-14T18:42:23-08:00
-draft: false
```
Comment (jeffhuyghebaert) [inline content/theme/characters/archetypes/republic/judicial.md:4]: need to keep this `draft: false`

File: content/theme/characters/archetypes/republic/judicial.md [hunk 2]
```diff
@@ -1,6 +1,36 @@
...
+layout: archetype
+attributes:
+awareness: 10
+   coordination: 9
+   knowledge: 8
+   physique: 10
+   presence: 7
+   reflex: 10
 
+ skills:
```
Comment (jeffhuyghebaert) [inline content/theme/characters/archetypes/republic/judicial.md]: This is indented one space too many, which is why the build is failing.

File: content/theme/characters/archetypes/republic/judicial.md [hunk 3]
```diff
@@ -1,6 +1,36 @@
...
+     strength: 3
+     stamina: 3
+     toughness: 1
+     run: 2
+   presence:
+     politics: 1
+     intimidate: 3
+   reflex:
+     melee: 2
+     acrobatics: 2
```
Comment (jeffhuyghebaert) [inline content/theme/characters/archetypes/republic/judicial.md]: brawl is probably more important here than acrobatics

**Review by jeffhuyghebaert — COMMENTED** [review]
Need to fix the indent to make the build complete

**Review by jetfire158 — COMMENTED** [review]
fixed

File: content/theme/characters/archetypes/republic/judicial.md [hunk 4]
```diff
@@ -2,5 +2,36 @@
 title: "Judicial"
 date: 2020-11-14T18:42:23-08:00
 draft: false
----
+layout: archetype
+attributes:
+awareness: 10
```
Comment (jeffhuyghebaert) [inline content/theme/characters/archetypes/republic/judicial.md]: this should be indented one space, and the ones below should all line up:
```
attributes:
 awareness: 10
 coordination: 9
 knowledge: 8
 physique: 10
 presence: 7
 reflex: 10

skills:
 awareness:
  track: 2
  search: 3
 coordinattion:
  pilot: 1
  shoot: 3
  drive: 2
 knowledge:
  security: 4
  investigate: 4
 physique:
  strength: 3
  stamina: 3
  toughness: 1
  run: 2
 presence:
  politics: 1
  intimidate: 3
 reflex:
  melee: 2
  brawl: 2

**Review by jetfire158 — COMMENTED** [review]
fixed

**Review by jeffhuyghebaert — APPROVED** [review]

**Pull request merged**